NETProvider/src/FirebirdSql.Data.UnitTests/FbCommandTests.cs

```diff
@@ -730,6 +756,15 @@ public void NoCommandPlanTest()
 				Assert.DoesNotThrow(() => { plan=cmd.CommandPlan; });
 				Assert.IsEmpty(plan);
 			}
+			}
+			finally
+			{
+				using (var cmd = Connection.CreateCommand())
+				{
+					cmd.CommandText = "drop table NoCommandPlanTest";
```

Comment:
```TearDown``` cares only about tables that are created in ```SetUp```.


I can move ```NoCommandPlan``` table to ```SetUp/TearDown``` but the table is used locally in this test.